tests/test_cli.py

```diff
@@ -75,8 +75,8 @@ def test_get(dispatcher_live_fixture, caplog):
     runner = CliRunner()
     result = runner.invoke(cli.cli, ['-u', dispatcher_live_fixture, 'get', '-i', 'empty'], obj={})
     assert result.exit_code == 0
-
-    assert "'prod_dict': {'dummy': 'empty_parameters_dummy_query', 'failing': '" in caplog.text
+    # TODO not sure it is supposed to be this way
```

Comment:
this depends on what is installed in the python environment. Presumably what's here should be what is installed in Actions. But it may differ from what you have locally.